documentation/optee_design.md

```diff
@@ -839,6 +839,28 @@ by early boot and passed to non-secure world are the following:
 Early boot DTB located in non-secure memory can be accessed by OP-TEE core
 only during its initialization, before non-secure world boots.
 
+## Early boot device tree overlay
+
+It is possible to provide a device tree overlay which contains the Early
```

Comment:
I think those 2 lines should be removed.

Reply:
I assume you mean
```
It is possible to provide a device tree overlay which contains the Early
boot device tree arguments as a standard device tree overlay structure.
```